src/frontend/src/components/DownloadMetadataTemplate/prepMetadataTemplate.ts

```diff
@@ -0,0 +1,189 @@
+
+const DATE_FORMAT = "YYYY-MM-DD";
+const BOOLEAN_FORMAT = "Yes/No";
+
+const TEMPLATE_HEADERS = [
+  // If position for sampleId changes, update `prepMetadataTemplate` func!
+  SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS.sampleId,
+  SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS.privateId,
+  SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS.publicId,
+  SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS.collectionDate,
+  SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS.collectionLocation,
+  SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS.sequencingDate,
+  SAMPLE_UPLOAD_METADATA_KEYS_TO_HEADERS.keepPrivate,
+];
+
+const TEMPLATE_HEADERS_EDIT = [
+  SAMPLE_EDIT_METADATA_KEYS_TO_HEADERS.currentPrivateID,
+  SAMPLE_EDIT_METADATA_KEYS_TO_HEADERS.newPrivateID,
+  SAMPLE_EDIT_METADATA_KEYS_TO_HEADERS.publicId,
+  SAMPLE_EDIT_METADATA_KEYS_TO_HEADERS.collectionDate,
+  SAMPLE_EDIT_METADATA_KEYS_TO_HEADERS.collectionLocation,
+  SAMPLE_EDIT_METADATA_KEYS_TO_HEADERS.sequencingDate,
+  SAMPLE_EDIT_METADATA_KEYS_TO_HEADERS.keepPrivate,
+];
+
+// We also use this elsewhere: if we see one of these uploaded, filter it out.
+export const EXAMPLE_SAMPLE_IDS = [
+  "Example Sample A",
+  "Example Sample B",
+  "Example Sample C",
+];
+
+export const EXAMPLE_CURRENT_PRIVATE_IDENTIFIERS = [
+  "example private ID A",
+  "example private ID B",
+  "example private ID C",
+];
+
+const EXAMPLE_ROWS = [
+  // Very first example row helps explain usage, but not fully valid.
+  [
+    EXAMPLE_SAMPLE_IDS[0], // sampleId
+    "Private sample name", // privateId
+    "(if available) GISAID ID", // publicId -- here as explainer
+    DATE_FORMAT, // collectionDate -- not valid, here as explainer in template
+    "North America/USA/California/Los Angeles County", // collectionLocation
+    DATE_FORMAT, // sequencingDate -- not valid, here as explainer in template
+    BOOLEAN_FORMAT, // keepPrivate -- not valid, here as explainer in template
+  ],
+  // Subsequent example rows are mostly valid, honest-to-goodness examples...
+  // ... except for the dates. This is to avoid Excel auto "correct".
+  [
+    EXAMPLE_SAMPLE_IDS[1], // sampleId
+    "id101", // privateId
+    "", // publicId -- optional, showing that with blank use
+    DATE_FORMAT, // collectionDate
+    "San Francisco County", // collectionLocation
+    "", // sequencingDate -- optional, showing that with blank use
+    "No", // keepPrivate
+  ],
+  [
+    EXAMPLE_SAMPLE_IDS[2], // sampleId
+    "id102", // privateId
+    "USA/CA-CZB-0001/2021", // publicId
+    DATE_FORMAT, // collectionDate
+    "North America/USA/California/San Francisco County", // collectionLocation
+    DATE_FORMAT, // sequencingDate
+    "No", // keepPrivate
+  ],
+];
+
+const SAMPLE_EDIT_INSTRUCTIONS = [
+  ["# Only fill out columns or cells you want to update the content"],
+  ["# Empty cells will not change what is currently in CZ GEN EPI database"],
+  ["# Fill cells with 'Delete' if you want to remove the existing content"],
+  ["# Save in .tsv .csv or .txt format"],
```

Comment:
there should be commas here

Reply:
taken from the copy again, i think the goal was to not have commas in case users want to import the tsv into csv.

Reply:
Yeah, the need for lack of commas was something I flagged. If they use it as a CSV and there's commas anywhere in the line boom boom. Similarly can't have tabs in the text, but we're not lunatics, so that one was never going to be an issue.